### PR TITLE
TSPS-359 update teaspoons e2e test to work with quota additions to service

### PR DESCRIPTION
**Pull request by jsotobroad**
This PR is meant to update the teaspoons e2e test to work with the new additions made to the service around the quota logic

### CODE REVIEW

File: e2e-test/tsps_gcp_e2e_test.py [hunk 1]
```diff
@@ -131,6 +131,25 @@ def poll_for_imputation_job(tsps_url, job_id, token):
 
     raise Exception(f"tsps pipeline did not complete in 25 minutes")
 
+def query_for_user_quota_consumed(tsps_url, token):
```
Comment (mmorgantaylor) [inline e2e-test/tsps_gcp_e2e_test.py:134]: don't have to change this in your PR, but at some point we should change these (and other similar) to `teaspoons_url`

File: e2e-test/tsps_gcp_e2e_test.py [hunk 2]
```diff
@@ -285,10 +304,24 @@ def add_member_to_terra_group(orch_url, group_name, email_address, role, token):
     }
     add_wdl_to_gcp_workspace(billing_project_name, workspace_name, wdl_namespace, wdl_name, method_definition_dict, root_entity_type, {}, {}, firecloud_orch_url, admin_token)
 
+    # create a new quota consumed method that teasspoons will run
```
Comment (mmorgantaylor) [inline e2e-test/tsps_gcp_e2e_test.py:307]: lol `teasspoons`

**Review by mmorgantaylor — APPROVED** [review]
exciting!

File: e2e-test/tsps_gcp_e2e_test.py (hunk 2 above)
Comment (jsotobroad) [reply e2e-test/tsps_gcp_e2e_test.py:307]: omfg, fixing!

File: e2e-test/tsps_gcp_e2e_test.py (hunk 1 above)
Comment (jsotobroad) [reply e2e-test/tsps_gcp_e2e_test.py:134]: yea lemme make a ticket thats blocked on the service renaming ticket to rename all this stuff - https://broadworkbench.atlassian.net/browse/TSPS-376

**Pull request merged**